english/marshmallow.md

```diff
@@ -103,6 +103,17 @@ The type of extreme mortal danger that so often accompanies confectionary inspir
 Quickly surveying your immediate surrounds, you notice a garishly dressed,
 vacant-eyed young man lounging on a nearby sofa.
 
+The man is not large but not small, but not average either. He is reciting poetry in a language you assume to be Egyptian, 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.